tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -934,6 +931,22 @@ def test_list_recommendations_with_changes_in_update_results(cluster_type: str):
                     memory_limit_entry = container_metrics[MEMORY_LIMIT_INDEX]
                     aggre_info = memory_limit_entry["results"]["aggregation_info"]
                     aggre_info["format"] = None
+            elif j == 109:
```

Comment:
Please rename the function to test_list_recommendations_notification_codes or something meaningful

Reply:
renamed it to `test_list_recommendations_notification_codes`. Thanks!